<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inductance_menu</class>
 <widget class="QWidget" name="inductance_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_r">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>110</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_h">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>143</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>h = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_h">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>143</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_L">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>176</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>L = </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_L">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>176</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_nu">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>44</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>µ = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nu">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>44</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_R">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>77</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_R">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>77</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>110</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>r = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_N">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_N">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>11</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="calculation_inductance">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>210</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить график</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
